src/amo/sagas/addonsByAuthors.js

```diff
@@ -0,0 +1,45 @@
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+import { SEARCH_SORT_POPULAR } from 'core/constants';
+import {
+  FETCH_OTHER_ADDONS_BY_AUTHORS,
+  NB_OTHERS_ADDONS_BY_AUTHORS,
+  loadOtherAddonsByAuthors,
+} from 'amo/reducers/addonsByAuthors';
+import searchApi from 'core/api/search';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+
+
+export function* fetchOtherAddonsByAuthors({ payload }) {
+  const { errorHandlerId, authors, slug, addonType } = payload;
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  yield put(errorHandler.createClearingAction());
+
+  try {
+    const state = yield select(getState);
+
+    const response = yield call(searchApi, {
+      api: state.api,
+      filters: {
+        addonType,
+        author: authors.join(','),
+        // We need one more add-on than the number to display because the API
```

Comment:
Is there a maximum number of add-ons the API will return, even if `page_size` is set quite large? It seems like this could require pagination...

Reply:
Not sure to understand. This would be a different feature to me. Here we just want to show a few other add-ons on the detail page. Displaying all the add-ons of a given author would rather use the search feature/UI.

Reply:
Ohhhhhhhh. Okay then yes, totally. I understand now. Ignore my pagination comments then, I'm daft.